frontend/src/views/Bridge.vue

```diff
@@ -622,4 +708,14 @@ export default Vue.extend({
 	box-shadow: 0 0 0 0 rgba(0, 0, 0, 1);
 	transform: scale(1);
 }
+.withdrawText{
+  font-size: clamp(12px, 1.25rem, 24px);
+}
+.withdrawSelect{
+  width: clamp(200px, 20vw, 300px);
+  margin:0 auto;
+}
+.withdrawBtn{
```

Comment:
If you want extra-clean css, you can add it like that:
.withdrawSelect{
  margin:0 auto;
}

.withdrawSelect,
.withdrawBtn{
  width:clamp(200px, 20vw , 300px);
}